doc/doxygen/Doxyfile.in

```diff
@@ -199,7 +199,7 @@ STRIP_FROM_INC_PATH    = @CF_OPENMS_DOCUMENTATION_STRIP_INCLUDES@
 # support long names like on DOS, Mac, or CD-ROM.
 # The default value is: NO.
 
-SHORT_NAMES            = NO
+SHORT_NAMES            = YES
```

Comment:
This will break links into class documentation AFAIR

Reply:
Ops, I put this commit on the wrong branch.  Removed.